lib/app/console_app.rb

```diff
@@ -49,6 +49,7 @@ def start_sync_now
       return unless connector_registered?
       puts 'Initiating synchronization NOW...'
       Core::ElasticConnectorActions.force_sync(connector_id)
+      puts "Successfully scheduled an on-demand sync for connector #{connector_id}"
```

Comment:
This message is a bit confusing.

```suggestion
      puts "Successfully synced for connector #{connector_id}"
```